server/spec/mutations/stacks/create_spec.rb

```diff
@@ -23,6 +23,31 @@
 
     it 'creates a new grid stack with parent' do
       grid
+      grid.stacks.create!(name: 'stack-parent')
```

Comment:
I'd still recommend removing this + the other `Stack.create!` in the other spec. The context doesn't actually match up with reality.

Reply:
Yeah, I can remove those.. although there is also a spec that checks that stack can be created before the parent exists.